python/pytest.ini

```diff
@@ -1,8 +1,10 @@
 [pytest]
 markers =
-  unit: marks unit tests
-  quality: marks quality tests
-  stress: marks stress tests
-  mg: marks a test as multi-GPU
-  memleak: marks a test as a memory leak test
+  unit: Quickest tests focused on accuracy and correctness
+  quality: More intense tests than unit with increased runtimes
+  stress: Longest running tests focused on stressing hardware compute resources
+  mg: Multi-GPU tests
+  memleak: Test that checks for memory leaks
+  no_bad_cuml_array_check: Test that should not check for bad CumlArray uses
```

Comment:
Is this one still needed?

Reply:
Its not. Removing the marker from `pytest.ini` will generate a warning for the tests that have this marker. I can update those tests but it seemed like that was starting to grow the scope of this PR. John wanted to keep it simple.

Reply:
I agree with keeping it simple, would you mind opening a gh issue to remove the marker? It would be a great first issue for anyone wanting to get familiar with the tests

Reply:
Done https://github.com/rapidsai/cuml/issues/3148